ReactWindows/ReactNative.Shared/Views/Text/TextDecorationLine.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Runtime.Serialization;
+
+namespace ReactNative.Views.Text
+{
+    /// <summary>
+    /// TextDecorationLine values.
+    /// </summary>
+    public enum TextDecorationLine
+    {
+        /// <summary>
+        /// Text has no line decoration.
+        /// </summary>
+        None,
+        /// <summary>
+        /// Text is Underlined.
+        /// </summary>
+        Underline,
+        /// <summary>
+        /// Text is Stroke out.
+        /// </summary>
+        LineThrough,
+        /// <summary>
+        /// Text is both Underlined and Stroke out.
+        /// </summary>
+        [EnumMember(Value = "underline line-through")]
+        UnderlineLineThrough
```

Comment:
We actually don't need this enum, you can just as easily switch case on the string itself.

---
In reply to: [104831419](https://github.com/ReactWindows/react-native-windows/pull/1056#discussion_r104831419) [](ancestors = 104831419)